config/config.properties

```diff
@@ -102,13 +102,30 @@ pravegaservice.dataLogImplementation=BOOKKEEPER
 
 # Maximum size (in bytes) for the Local Shared Cache (shared by all Segment Containers on this Segment Store instance).
 # Valid values: Positive integer.
-# Recommended values: Multiples of 1GB. Choosing a lower size will conserve memory and disk resources at the expense of
-# a lot of cache misses (resulting in Tier2 reads and possibly high latencies). Choosing a higher size will keep data
-# in the cache for longer, but larger values may not be appropriate for all Cache implementations. For an In-Memory cache,
-# the Java Heap will fill up very quickly, possibly causing the process to crash. For a Memory-Disk hybrid cache (such as
-# RocksDB) a larger value will be appropriate, as that cache implementation should be able to offload extra memory to disk
-# and fetch it back when needed.
-#pravegaservice.cacheMaxSize=17179869184
+# Recommended values: Multiples of 1GB, but less than XX:MaxDirectMemorySize. Choosing a lower size will conserve memory
+# at the expense of a lot of cache misses (resulting in Tier2 reads and possibly high latencies). Choosing a higher size
+# will keep data in the cache for longer, but at the expense of memory. The Segment Store uses a Direct Memory Cache backed
+# by direct ByteBuffers, so if XX:MaxDirectMemorySize is not set to a value higher than this one, the process will
+# eventually crash with an OutOfMemoryError.
+#pravegaservice.cacheMaxSize=4294967296
+
```

Comment:
Yes, it would be. However I would prefer to change that in a different PR since there's a number of other things that set this externally (such as the K8s operator).